volume/volume.go

```diff
@@ -308,6 +309,12 @@ type Enumerator interface {
 	SnapEnumerate(volID []string, snapLabels map[string]string) ([]*api.Volume, error)
 }
 
+// Water provides a set of function to get volume
+type Watcher interface {
+	// Gets Volume notifier
+	GetVolumeWatcher(locator *api.VolumeLocator, labels map[string]string) (chan *api.Volume, error)
```

Comment:
As discussed with Aditya offline, this might not be necessary. Looking into ways to remove this function.

Reply:
First pass on the code, I think it will be really hard to manage the channel passing from top to sdkServer and protoDriver. 

Another way to send the value is through channel, but it becomes hard to manage too when we throw business logic into contexts. 

@adityadani let me know what you think.

Reply:
Lets keep this interface and function in that case.